### PR TITLE
ci(GitHub Actions): Migrate from CircleCI to GitHub Actions [CICD-236]

### PR DESCRIPTION
**Pull request by AodhanLP**

### CODE REVIEW

**Review by ryasmi — APPROVED** [review]
Looks okay to me 👍  Will wait for @wobrien's approval on this first one.

**Review by christopher-learningpool — APPROVED** [review]

**Comment by christopher-learningpool** [comment]
Hold off on Merging until @wobrien can break the Circle CI connection.

**Review by ryasmi — APPROVED** [review]

**Pull request merged**
